configs/setups/awiesm/awiesm.yaml

```diff
@@ -358,6 +358,8 @@ fesom:
 
         namelist_changes:
                 namelist.config:
+                        ale_def:
+                                which_ale: "${ALE_scheme}"                        
```

Comment:
Is this something new to FESOM 2.5 or 2.6 or was it always in 2.0 and 2.1?

Reply:
Does the parameters defined below still hold?

```yaml
        # ALE_scheme switch
        # -----------------
        # Settings for Gent/McWilliams based upon the selected ALE scheme.
        choose_ALE_scheme:
            linfs:
                K_gm_max: "remove_from_namelist"
                K_hor: 5000.0
                K_gm: 5000.0
            zstar:
                K_gm_max: 2000.0
                K_hor: 3000.0
                K_gm: remove_from_namelist
            "*":
                is: invalid
```

Reply:
Good point! `K_hor` should go into `namelist.tra/tracer_phys`, `K_gm_max` is fine I think, and `K_gm` is not needed anymore apparently. But I am unsure about the `K_gm_max` setting for `linfs`. Is it `3000` then?

Maybe @JanStreffing or @koldunovn can comment on that?

Reply:
Default GM value differs between FESOM2 core2 standalone (2000) and FESOM2 core2 in AWI-CM3 (3000). The retuning from 3000 to 2000 was found to be beneficial for the standalone model while Nikolay worked on the 2.1. release. I tested both for AWI-CM3.1 and found 3000 produced lower biases.

Reply:
Right, sorry!
`    k_gm_max = 3000`